src/biokbase/narrative/tests/test_job.py

```diff
@@ -904,6 +936,20 @@ def test_in_cells__batch__diff_cells(self):
 
         self.assertFalse(batch_job.in_cells(["goodbye", "hasta manana"]))
 
+    @mock.patch(CLIENTS, get_mock_client)
+    def test_batch_cell_ids__not_batch(self):
+        job = Job.from_job_id(JOB_COMPLETED)
+        self.assertEqual(job.batch_cell_ids(), None)
+
+    @mock.patch(CLIENTS, get_mock_client)
+    def test_batch_cell_ids__batch_job(self):
+        batch_fam = get_batch_family_jobs(return_list=True)
+
+        job = batch_fam[0]
+        self.assertEqual(
+            set(job.batch_cell_ids()), set([CELL_ID_LIST[2], CELL_ID_LIST[3]])
+        )
+
```

Comment:
Tests for the new `batch_cell_ids()` function -- I've edited the batch job to have children in two different cells.

Reply:
Tricky! No reason it shouldn't happen, though.